index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>José Fernández-Caminero's Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <img class="artdeco1" src="resources/image_geometry_1.svg" alt="Geometrical decoration">
+            <img src="resources/ic_menu.svg" alt="Toolbar menu">
+        </nav>
+    </header>
+
+    <section>
+        <h1 class="textdeco1 text1width">José Fernández-Caminero</h1>
+
+        <p class="crazyParagraph">I love Full-Stack Engineering and I can code websites that are both the most complex and the most efficient.</p>
+
+        <button>Start collaboration</button>
```

Comment:
- There are a few differences between your design and the Figma design template, this button takes up more width in the design template and is also aligned in the center. Please try to fix this 🙏 . Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your Project 👇:
<img width="156" alt="jose button" src="https://user-images.githubusercontent.com/73607512/198153441-eb4e0e65-e3b4-4309-92cd-1440bdb0c0cf.PNG">

Figma template:
<img width="254" alt="jose button figma" src="https://user-images.githubusercontent.com/73607512/198153554-9ad27ada-075c-4c93-940f-917c23856d15.PNG">

Reply:
@mckent05 I'm sorry, but they seem exactly the same. 288px. Did you zoom in accidentally?

Reply:
Hi @fdezcaminero,

As a result of the request for a second opinion, I'm currently reviewing your pull request.

Reply:
@medaminedev66 Thanks! Remember, I'm not saying that the requests of @mckent05 are incorrect. It's just that I bumped into a problem that seems very tricky to me. Maybe he himself did not anticipate how tricky it was. I posted at Stack Overflow yesterday and still no one's answer has worked:

https://stackoverflow.com/questions/74228659/give-a-resizable-background-image-a-full-width